src/components/middle/immedia/constants.ts

```diff
@@ -1,9 +1,10 @@
-import { DEBUG } from '../../../config';
+// import { DEBUG } from '../../../config';
 
-const WEBSOCKET_URL = DEBUG
-  ? 'http://localhost:3000/ws'
-  : 'http://immedia.herokuapp.com/ws';
-// const WEBSOCKET_URL = "http://immedia.herokuapp.com/ws";
+// uncomment for production
```

Comment:
I believe we can set `APP_ENV` env var to `production` to avoid the uncomment.

Reply:
Done in 2f6cb40de8bb82305d24d12e807ecd047bb165c5